tensorflow/core/profiler/utils/xplane_schema.cc

```diff
@@ -221,6 +221,7 @@ const StatTypeMap& GetStatTypeMap() {
       {"memory_size", kDevCapMemorySize},
       {"compute_cap_major", kDevCapComputeCapMajor},
       {"compute_cap_minor", kDevCapComputeCapMinor},
+      {"dev_manufacturer", kDevManufacturer},
```

Comment:
Let's not abbreviate this one and go with: "device_manufacturer"

Reply:
Renamed it to "device_vendor"